docs/cloud/setup-azure.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Setting up lakeFS Cloud on Azure
+title: On Azure
```

Comment:
IMO the previous title is more descriptive compared to this one.

Reply:
This is only the title in the navigation menu, I think it makes more sense to keep it short and avoid redundancy:
![image](https://user-images.githubusercontent.com/3149885/231768724-4e3cc1f4-5c64-4ba0-b8b1-f283b8018745.png)

WDYT?